### PR TITLE
scss cleanup

### PR DESCRIPTION
**Pull request by davidrapson**
- Extract cads-spacers into storybook styles
- Remove cads-spacing-0
- Remove unused cads-interactive-target-size__small
- Move forms styles to components
- Remove old cads-iconfont partial (moved to generics)

### CODE REVIEW

File: scss/6-components/_header.scss [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // ============================================================================
-// Footer
+// Header
```
Comment (baseonmars) [inline scss/6-components/_header.scss:2]: :)

File: styleguide/styles.scss [hunk 2]
```diff
@@ -62,6 +62,19 @@
   }
 }
 
+$cads-spacers: (
+  (
+    0: 0,
```
Comment (baseonmars) [inline styleguide/styles.scss:67]: `$cads-spacing-0` isn't used by any macros etc?

**Review by baseonmars — APPROVED** [review]
Looks fine - I'm gonna assume you're confident in removing cads-spacing-0, if not I don't see the harm in leaving it it.

File: styleguide/styles.scss (hunk 2 above)
Comment (davidrapson) [reply styleguide/styles.scss:67]: It's never used. Plus having a variable name for 0 is a bit much. The Mayans already sorted this for us 🙃

Comment (baseonmars) [reply styleguide/styles.scss:67]: I figured it'd be handy in a loop, but if it's not used and brings us no joy. then 🧹. What year is the world meant to end again? ;)

**Pull request merged**